### ISSUE TITLE
[Feature] Keep Header and Filter Sidebar Fixed for Improved User Experience

### ISSUE THREAD

**Issue by himanshumishra1309** [Yashgabani845/hiring-portal#355]
**Description:** Currently, **both the header and filter sidebar scroll away when navigating** through the job listings. This **causes inconvenience**, as users lose access to the filters and top navigation options, making it harder to switch filters or navigate back to other sections. **Fixing this will enhance usability, especially on long pages with multiple job postings**.

Before scrolling: 
![Screenshot 2024-11-09 131938](https://github.com/user-attachments/assets/fb05bf69-04e2-4316-b80a-25f3b4b53a46)

After Scrolling: 
![Screenshot 2024-11-09 131952](https://github.com/user-attachments/assets/a07e08b6-5c46-40ed-9334-7b3212350e66)

**Proposed Solution:**

**Fixed Header:** The header should remain fixed at the top of the page to allow easy access to navigation links and the login button, regardless of scroll position.

**Fixed Filter Sidebar:** The filter sidebar should stay in place on the left side of the screen, allowing users to update filters as they scroll through job listings. Only the job listing area on the right side should be scrollable.

I am experienced with the technologies used in this project and skilled in the MERN stack, making me well-suited to address both frontend and backend issues. I would be more than happy to contribute fixes for these UI issues. If the maintainers are open to it, I can start by addressing one or all of these problems, ensuring that the solutions align with the project’s overall design and functionality standards.

If you would like me to submit a pull request with these fixes, please let me know. I am open to feedback on my proposed solutions and would appreciate any guidance on project-specific coding standards or best practices.

Additionally, if you need assistance with backend development or other aspects of the project, please feel free to reach out to me. Here is my LinkedIn profile for contact: [www.linkedin.com/in/himanshu-mishra-459b882b4](https://github.com/samyakmaitre/eventmint/issues/www.linkedin.com/in/himanshu-mishra-459b882b4).

**Comment by Yashgabani845** [comment]
@himanshumishra1309 can you please clone repo and check local latest code and then raise the issue current code is lot more chnaged than the deployed old one

**Comment by himanshumishra1309** [comment]
After testing the latest code in the local environment, I noticed that while the header is now fixed, the filter sidebar still scrolls with the page. For a better user experience, it would be beneficial if the filter sidebar remained fixed on the left side of the screen, allowing users to adjust filters as they browse through job listings. This way, only the job listing section on the right would be scrollable, making it easier for users to view and apply filters without losing their position on the page.

Before scrolling: 
![Screenshot 2024-11-10 131234](https://github.com/user-attachments/assets/e4e4035e-488b-4b21-bb66-95e865d1b96a)


After scrolling: 
![Screenshot 2024-11-10 131248](https://github.com/user-attachments/assets/7c373df3-2a0b-4a20-9ab3-74ad9c1b59cb)


**Proposed Solution:**

- Make the filter sidebar fixed on the left side of the screen.
- Allow only the job listings section on the right to be scrollable.

**Benefit:** This change will improve navigation and accessibility, providing a smoother experience for users when browsing and applying filters to job listings.
